frappe/desk/doctype/notification_log/notification_log.py

```diff
@@ -88,10 +89,21 @@ def get_email_header(doc):
 	}[doc.type or 'Default']
 
 
+def get_truncated_title(doctype, docname, title_field=None, truncate_length=None):
```

Comment:
Truncate title with CSS not Python. The full title should still be stored in the Notification Log. It should truncate just in view.